index.html

```diff
@@ -30,67 +30,69 @@
     <div class="d-flex flex-column flex-wrap">
       <div class="m-auto pt-3 pb-4">
         <br />
-        <button type="button" class="btn btn-danger" id="choose-red-player">Red</button>
+        <button type="button" class="btn btn-danger player-btn">Player 1</button>
+        <button type="button" class="btn btn-outline-danger player-btn d-none">Player 1</button>
         &nbsp;
-        <button type="button" class="btn btn-outline-warning" id="choose-yellow-player">Yellow</button>
+        <button type="button" class="btn btn-warning player-btn d-none">Player 2</button>
+        <button type="button" class="btn btn-outline-warning player-btn">Player 2</button>
         &nbsp;&nbsp;|&nbsp;&nbsp;
-        <button type="button" class="btn btn-dark" id="reset">Reset Board</button>
+        <button type="button" class="btn btn-dark" id="reset">New Game</button>
       </div>
       <br />
       <div class="container grid mx-auto text-center px-5 pt-4">
         <div class="row justify-content-center">
-          <div class="circle">0</div>
-          <div class="circle">1</div>
-          <div class="circle">2</div>
-          <div class="circle">3</div>
-          <div class="circle">4</div>
-          <div class="circle">5</div>
-          <div class="circle">6</div>
+          <div class="circle"></div>
```

Comment:
One way to keep track of the columns would be to use data attributes: https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes